blog/drafts/ultimate-commands-guide/ultimate-commands-guide.md

```diff
@@ -0,0 +1,313 @@
+# Ultimate validator maintenance guide. Testnet: Euler-1.
+
+## Glossary
+
+  **Bandwidth** - 1 bandwidth of approximately 55 600 000 cyb. The recovered unit of your account. Depends on your cyb count. Used to complete transactions in the cyberd blockchain. Transaction costs from 8 to 800 bandwidth - recalculate by price sliding window. Full regeneration time 24 hours
+
+  **commission** -  tokens that you've earned with validation. You can take them at any time.
+
+  **illiquid tokens** - non-transferable tokens that you've delegated to the validator. Delegation process duration - 1 block. **Unbonding** process, or taking back share - 3 weeks.
+
+  **link** - reference between CID key and CID value. ransaction costs from 8 to 800 bandwidth - recalculate by price sliding window. Linking time 100 s.
+
+  **liquid tokens** - transferable tokens in cyberd blockchain
+
+  **local keystore** - store with keys in you local machine
+
+  **rewards** - tokens that you've earned with delegation. You can take them at any time.
+
+  **<comission_rate_percentage>** - must be fraction >0 and <=1
+
+  **<delegator_address>** - delegator address. Starts with `cyber` most often coinciding with **<key_address>**
+
+  **<key_address>** - account address. Starts with `cyber`
+
+  **<key_name>** - name of account in cybercli
+
+  **<operator_address>** - validator address. Starts with `cybervaloper`
+
+  **<shares_percentage>** - must be fraction >0 and <=1
+
+  **<testnet_chain_id>** - version of testnet.
+
+## General commands
+
+##### Show all validators
+Return set of all active and jailed validators.
+```bash
+docker exec cyberd cyberdcli query staking validators --trust-node
+```
+
+##### Show chain status
+Return general chain information
+```bash
+docker exec cyberd cyberdcli status --indent
+```
+
+##### Distribution params
+```bash
+docker exec cyberd cyberdcli query distr params --trust-node
+```
+
+##### The amount of outstanding rewards
+Return the sum of rewards in a pool
+```bash
+docker exec cyberd cyberdcli query distr outstanding-rewards --trust-node
+```
+
+##### Staking params
+Chain staking info
+```bash
+docker exec cyberd cyberdcli query staking params --trust-node
+```
+
+##### Staking pool
+```bash
+docker exec cyberd cyberdcli query staking pool --trust-node
+```
+
+## Account management
+
+##### Import an account by seed phrase and store it in local keystore
+```bash
+docker exec -ti cyberd cyberdcli keys add <your_key_name> --recover
+```
+
+##### Import an account by ETH private key and store it in local keystore
```

Comment:
not only by ETH private key. Just by private key. And of course it could be your ETH  private key